src/Interpreters/TreeRewriter.h

```diff
@@ -90,6 +90,7 @@ struct TreeRewriterResult
     void collectSourceColumns(bool add_special);
     void collectUsedColumns(const ASTPtr & query, bool is_select, bool visit_index_hint);
     Names requiredSourceColumns() const { return required_source_columns.getNames(); }
+    Names sourceColumns() const { return source_columns.getNames(); }
```

Comment:
where is this method used? my plain ctrl+f in browser cannot find any other usage :)

Reply:
ah sorry, I was working on a different approach and forgot to remove this. Removed it now.